app/components/object-inspector/property.js

```diff
@@ -28,11 +28,11 @@ export default Component.extend({
 
   isObject: equal('valueType', 'type-object'),
 
-  isInstance: equal('valueType', 'type-instance'),
```

Comment:
Is there a reason we are removing `instance`?

Reply:
instance was specific to Ember.typeOf, its now also just an `object`. the only place where it was used is in the `canDig` function

Reply:
Okay, I was worried if we didn't have the instance, we would not be able to click further into the instances. You're saying it works still with this change?

Reply:
right, it still works